pkg/testsuites/standard_suites.go

```diff
@@ -259,8 +259,9 @@ var staticSuites = []ginkgo.TestSuite{
 			}
 			return strings.Contains(name, "[Suite:openshift/network/ipsec")
 		},
-		Parallelism: 1,
-		TestTimeout: 120 * time.Minute,
+		Parallelism:                1,
+		TestTimeout:                60 * time.Minute,
+		ClusterStabilityDuringTest: ginkgo.Disruptive,
```

Comment:
please elaborate on why this is disruptive to a cluster?  With the flow described by Tim, I'd expect workloads to continue running successfully.

Reply:
yes, this has to be removed as well now, updated the PR.